src/main/java/roomescape/infrastructure/LoginMemberArgumentResolver.java

```diff
@@ -0,0 +1,41 @@
+package roomescape.infrastructure;
+
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.core.MethodParameter;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+import roomescape.member.LoginMember;
+import roomescape.member.Member;
+import roomescape.member.MemberService;
+
+@Component
+public class LoginMemberArgumentResolver implements HandlerMethodArgumentResolver {
+    private MemberService memberService;
+    private JwtTokenUtil jwtTokenUtil;
```

Comment:
`final`로 처리하는 것이 더 좋아보여요!
[[Java] final 을 알아보자
](https://yeoonjae.tistory.com/entry/Java-final-%EC%9D%84-%EC%95%8C%EC%95%84%EB%B3%B4%EC%9E%90)

```suggestion
    private final MemberService memberService;
    private finla JwtTokenUtil jwtTokenUtil;
```

Reply:
수정했습니다!